test/services/omise_payment_service_test.dart

```diff
@@ -4,6 +4,7 @@ import 'package:omise_flutter/src/pages/select_payment_method_page.dart';
 import 'package:omise_flutter/src/pages/payment_authorization_page.dart';
 import 'package:omise_flutter/omise_flutter.dart';
 import 'package:webview_flutter/webview_flutter.dart';
+// ignore: depend_on_referenced_packages
```

Comment:
Just wonder  this is safe to ignore right?

Reply:
This is just to ignore the warning as I am using some internal class to verify my unit tests and this class is not exposed by the package that i am using directly.

Reply:
📓 Noted